debian/usr.bin.snap-confine

```diff
@@ -112,6 +114,15 @@
     mount options=(rw bind) /snap/ubuntu-core/*/libx32/ -> /libx32/,
     mount options=(rw bind) /snap/ubuntu-core/*/lib64/ -> /lib64/,
     mount options=(rw bind) /snap/ubuntu-core/*/usr/ -> /usr/,
+    mount options=(rw bind) /snap/ubuntu-core/*/etc/alternatives/ -> /etc/alternatives/,
+
+    # This is a somewhat of a blank check for mount profiles.  Mount profiles
+    # are written by the trusted snapd to a root-writable location and
+    # processed by snap-confine. Regardless of configuration options and
+    # runtime factors they are processed after pivot_root (on classic systems)
+    # so snap.rootfs is not a factor.
+    mount options=(rw bind),
+    mount options=(ro bind),
```

Comment:
CC @tyhicks 
I very much do not like these mount rules. The problem is not about who is writing the mount profiles, it is about attacks against snap-confine and privilege escalation. These rules as written provide virtually no protection against privilege escalation if an attacker can gain control of snap-confine. Can you give specific examples why these rules are required and not more-specific rules that are coordinated between snap-confine and snapd?

Also, you didn't address my questions in this comment:
"Doesn't snapd control what the mount points are via interfaces? As such, we are still in control of the mountpoints within the snapcore project.

Even if not, aren't the mount for things to be mounted inside the snap? As such, we should be able to limit the target directory at the very least. For example, it would be exceedingly unfortunate if we mounted /var as rw in the snap writable area since then the snap has control to modify /var/lib/snapd/....

With that in mind, I'm now looking at these mount rules and wondering why many of them are 'rw'-- eg, everything under 'when --enable-rootfs-is-core-snap is NOT used' and nvidia."